### PR TITLE
Fix forms for various Gen 4 pokemon

### PR DESCRIPTION
**Pull request by chuckleslove**

### CODE REVIEW

File: data/pokemon.json [hunk 1]
```diff
@@ -1,445 +1,534 @@
...
+  "483_00": "Dialga",
+  "484_00": "Palkia",
+  "485_00": "Heatran",
+  "486_00": "Regigigas",
+  "487_11": "Giratina",
+  "487_12": "Giratina",
+  "488_00": "Cresselia",
+  "489_00": "Phione",
+  "491_00": "Darkrai",
+  "492_11": "Shaymin"
```
Comment (123FLO321) [inline data/pokemon.json]: Add all forms with form names for future use.

**Review by 123FLO321 — CHANGES_REQUESTED** [review]

**Review by chuckleslove — COMMENTED** [review]
Added form names

**Comment by chuckleslove** [comment]
Added form names for Gen 4 stuff already in the game files.

**Review by chuckleslove — COMMENTED** [review]
Forms for Gen 4 renamed to match previous formatting.

**Review by chuckleslove — COMMENTED** [review]
Added Gen 4 pokemon IDs to functions.php

**Comment by chuckleslove** [comment]
functions.php updated as well for Gen 4 pokemon.

**Review by 123FLO321 — APPROVED** [review]

**Pull request merged**